config/job_conf.xml

```diff
@@ -13,9 +13,9 @@
 	    <param id="k8s_namespace">default</param>
 	    <param id="k8s_supplemental_group_id">0</param>
 	    <param id="k8s_fs_group_id">0</param>
-	    <param id="k8s_pull_policy">IfNotPresent</param>
```

Comment:
Please merge the latest develop branch available to your feature branch, to resolve the conflict for the config file.